src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+	
+	private String value;
```

Comment:
You might want to use a more specific name other than `value`. Something like `_block` would be better.